vendor/processors/stripe_processor.py

```diff
@@ -441,7 +456,8 @@ def get_vendor_customers_in_stripe(self, customer_email_list, site):
         Returns all vendor customers who have been created as Stripe customers
         """
         users = CustomerProfile.objects.filter(
-            user__email__iregex=r'(' + '|'.join(customer_email_list) + ')',  # iregex used for case insensitive list match
+            user__email__iregex=r'(' + '|'.join(customer_email_list) + ')', # iregex used for case insensitive list match
+            meta__has_key='stripe_id',
```

Comment:
What happens if a customer exists in vendor but for some reason does not have the stripe_id?

Reply:
Then it will be returned in the next exclude function, and a stripe customer will be created and the stripe id saved to its meta

Reply:
Glad we could figure out the edge cases togther. Nice work